MdeModulePkg/Core/Dxe/Image/Image.c

```diff
@@ -1725,7 +1725,10 @@ CoreStartImage (
   // Image has completed.  Verify the tpl is the same
   //
   ASSERT (Image->Tpl == gEfiCurrentTpl);
-  CoreRestoreTpl (Image->Tpl);
+  if (Image->Tpl != gEfiCurrentTpl) {
+    // MU_CHANGE - reduce TPL restore

```

Comment:
The `MU_CHANGE` tag here should be outside the new if statement introduced.

Reply:
Fixed.